File: C,C++/p10696.c
#include <stdio.h>

int f91(int num) {
    if (num >= 101) return num-10;
    else return 91;
}

int main () {
    while (1) {
        int num;
        scanf("%d", &num);
        if (num == 0) break;
        printf("f91(%d) = %d\n", num, f91(num));
    }
    return 0;
}

File: Java/p10696.java
import java.util.Scanner;

public class p10696 {
	public static void main (String [] args) {
		Scanner input = new Scanner(System.in);
		
		while (true) {
			int num = input.nextInt();
			
			if (num == 0) break;
			
			System.out.printf("f91(%d) = %d\n", num, f91(num));
		}
	}
	
	static int f91(int num) {
		if (num <= 100) return f91(f91(num+11));
		else return num-10;
	}
}